app/models/user.rb

```diff
@@ -87,8 +87,17 @@ def archive!
   def self.from_omniauth(auth)
     user = where(provider: auth.provider, uid: auth.uid).first_or_create do |u|
       u.name = auth[:info][:name]
+      u.email = auth[:info][:email]
+      u.avatar_thumb_url = auth[:info][:avatar_thumb_url]
+      u.birthday = auth[:info][:birthday]
     end
     user.update_role(auth[:info][:groups])
+    user.update(
```

Comment:
These attributes were only synced when creating the user, not when logging in again